content/en/security/application_security/getting_started/serverless.md

```diff
@@ -0,0 +1,142 @@
+   For **Go functions only** also add:
+   ```yaml
+   DD_UNIVERSAL_INSTRUMENTATION:
+     environment:
+       DD_UNIVERSAL_INSTRUMENATION: true
+   ```
+3. Redeploy the function and invoke it. After a few minutes, it appears in [ASM views][3].
+
+[1]: /serverless/serverless_integrations/plugin
+[2]: /serverless/libraries_integrations/extension
+[3]: https://app.datadoghq.com/security/appsec?column=time&order=desc
+
+{{% /tab %}}
+
+{{% tab "Custom" %}}
+
+1. Install the Datadog tracer:
+   - **Java**: [Configure the layers][1] for your Lambda function using the ARN in one of the following formats, depending on where your Lambda is deployed. Replace `<AWS_REGION>` with a valid AWS region such as `us-east-1`:
+     ```sh
+     # In AWS commercial regions
+     arn:aws:lambda:<AWS_REGION>:464622532012:layer:dd-trace-java:8
+     # In AWS GovCloud regions
+     arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:dd-trace-java:8
+     ```
+   - **Go**: The Go tracer doesn't rely on a layer and is a regular Go module. You you can upgrade to its latest version with:
+     ```sh
+     go get -u github.com/DataDog/datadog-lambda-go
+     ```
+   - **.NET**: [Configure the layers][1] for your Lambda function using the ARN in one of the following formats, depending on where your Lambda is deployed. Replace `<AWS_REGION>` with a valid AWS region such as `us-east-1`:
+     ```sh
+     # x86-based Lambda in AWS commercial regions
+     arn:aws:lambda:<AWS_REGION>:464622532012:layer:dd-trace-dotnet:6
+     # arm64-based Lambda in AWS commercial regions
+     arn:aws:lambda:<AWS_REGION>:464622532012:layer:dd-trace-dotnet-ARM:6
+     # x86-based Lambda in AWS GovCloud regions
+     arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:dd-trace-dotnet:6
+     # arm64-based Lambda  in AWS GovCloud regions
+     arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:dd-trace-dotnet-ARM:6
+     ```
+2. Install the Datadog Lambda Extension by configuring the layers for your Lambda function using the ARN in one of the following formats. Replace `<AWS_REGION>` with a valid AWS region such as `us-east-1`:
+   ```sh
+   # x86-based Lambda in AWS commercial regions
+   arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:36
+   # arm64-based Lambda in AWS commercial regions
+   arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension-ARM:36
+   # x86-based Lambda in AWS GovCloud regions
+   arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:Datadog-Extension:36
+   # arm64-based Lambda in AWS GovCloud regions
+   arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:Datadog-Extension-ARM:36
+   ```
+
+3. Enable ASM by updating your `serverless.yml` (or whichever way you set environment variables for your function):
+   ```yaml
+   environment:
+     DD_SERVERLESS_APPSEC_ENABLED: true
+   ```
+   For **Go functions only** also add:
+   ```yaml
+   DD_UNIVERSAL_INSTRUMENTATION:
+     environment:
+       DD_UNIVERSAL_INSTRUMENATION: true
+   ```
```

Comment:
Add a section for Node and Python environment variables:

For **Node or Python** functions only, please add:
```yaml
    environment:
        DD_EXPERIMENTAL_ENABLE_PROXY: true
```